README.md

```diff
@@ -442,15 +540,15 @@ drive push -files a1/b2
 drive push -directories tf1
 ```
 
-Like most commands [.driveignore](#driveignore) can be used to filter which files to push.
+Like most commands [.driveignore](#excluding-and-including-objects) can be used to filter which files to push.
 
 Here is an example using drive to backup the current working directory. It pushes a tar.gz archive created on the fly. No archive file is made on the machine running the command, so it doesn't waste disk space.
 
 ```shell
 tar czf - . | drive push -piped backup-$(date +"%m-%d-%Y-"%T"").tar.gz
 ```
 
-+ Note:
++ Notes:
```

Comment:
Please keep it as 'Note', 'Notes' are usually added at the end of the document or a section. This one is a `Note` as in `NB`(Nota bene)` without a prefixing title.

Reply:
Here this mite help Love GOD 1st for he loved us Amen!!!!